pkg/aggregator/demultiplexer.go

```diff
@@ -896,9 +887,9 @@ func (d *ServerlessDemultiplexer) Serializer() serializer.MetricSerializer {
 	return d.serializer
 }
 
-// Aggregator returns the main buffered aggregator
+// Aggregator returns nil since the Serverless Agent doesn't run an Aggregator.
```

Comment:
Would it be possible to remove this method entirely?

Reply:
Unfortunately this is still conditional between the main Demux and the Serverless one (in the DogStatsD server instantiation mainly), I have yet to find the correct separation here for it to not be part of the `Demultiplexer` interface anymore. I've tried for half an hour and did not succeed easily, I'm creating a follow-up card but not I won't do it here in this PR if you don't mind.